fastai/data/core.py

```diff
@@ -9,7 +9,14 @@
 
 # Cell
 @typedispatch
-def show_batch(x, y, samples, ctxs=None, max_n=9, **kwargs):
+def show_batch(
+    x, # Input(s) in the batch
+    y, # Target(s) in the batch
+    samples, # List of (`x`, `y`) pairs of length `max_n`
+    ctxs=None, # List of `ctx` objects to show data. Could be matplotlib axis, DataFrame etc
+    max_n=9, # Maximum number of `samples` to show
```

Comment:
```suggestion
    max_n:int=9, # Maximum number of `samples` to show
```

Reply:
Adding type hint for max_n, failed a few check jobs. Failing check jobs in [5601bae](https://github.com/muellerzr/fastai-docment-sprint/pull/93/commits/5601baefc56a4e9e63e5ac7a5e92b2898503ee8d)